code/modules/antagonists/blob/overmind.dm

```diff
@@ -97,7 +97,7 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 
 /mob/camera/blob/proc/is_valid_turf(turf/T)
 	var/area/A = get_area(T)
-	if((A && !(A.area_flags & BLOBS_ALLOWED)) || !T || !is_station_level(T.z) || isspaceturf(T))
+	if((A && !(A.area_flags & BLOBS_ALLOWED)) || !T || !is_station_level(T.z) || isspaceturf(T) || istype(T, /turf/open/openspace))
```

Comment:
doesn't that change the logic behind blob expanding as well?
It is allowed to exand on openspace

Reply:
Openspace imo should not be expandable

Reply:
Anyway no it doesn't, I tried expanding onto a staircase top part and it worked fine.

Reply:
This proc is used to determine valid locations
- To "score" for the blob (their critical mass)
- To move onto / place a core onto during the pre-placement phase
- To move the core with Move Core action
It *does not* dictate
- Where the blob can expand
- Where the blob can move to post-core placement